src/Network/Pusher/Data.hs

```diff
@@ -151,3 +165,108 @@ type Event = T.Text
 type EventData = T.Text
 
 type SocketID = T.Text
+
+data Notification = Notification
+  { interests :: Interest
```

Comment:
Named that way originally to reflect the field in the JSON object, but I absolutely agree it's confusing. Changed.